src/Nox.Reference.GetData/CliCommands/EnviromentSetupCommand.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+
+namespace Nox.Reference.GetData.CliCommands;
+
+public class EnviromentSetupCommand : ICliCommand
```

Comment:
TODO: just to think do we really need such command. It's enough to create folders in the main flow in CliCommandExecutor.